internal/boxcli/init.go

```diff
@@ -32,10 +32,6 @@ func runInitCmd(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	box, err := devbox.Open(path, cmd.ErrOrStderr())
-	if err != nil {
-		return errors.WithStack(err)
-	}
 
-	return errors.WithStack(box.GenerateEnvrc(false, "init"))
+	return nil
```

Comment:
nit, above if statement can be removed and this can just be `return errors.WithStack(err)`